spring-data-cassandra/src/test/java/org/springframework/data/cassandra/test/integration/template/async/AsynchronousCassandraTemplateTest.java

```diff
@@ -58,10 +53,151 @@
 @ContextConfiguration
 public class AsynchronousCassandraTemplateTest extends AbstractSpringDataEmbeddedCassandraIntegrationTest {
 
-	@Configuration
-	public static class Config extends IntegrationTestConfig {
+	@Autowired CassandraOperations cassandraOperations;
+
+	@Before
+	public void before() {
+		deleteAllEntities();
+	}
+
```

Comment:
Remove the word "test" from all actual (`@Test` annotated) test case method names.